src/main/java/Model/Web/Users.java

```diff
@@ -0,0 +1,22 @@
+package Model.Web;
+
+import com.google.gson.annotations.Expose;
+
+import java.util.List;
+
+public class Users {
```

Comment:
ak tu nic nepribudne tento model redundant, pouzivaj iba List<User> a vystacis si stym. Nikdy nenazvyvaj model xxxs, to je proste list, nova trieda znamena nejaky nadobjekt ktory zastresuje 1 User alebo list